src/lib/coda_base/command_transaction.ml

```diff
@@ -0,0 +1,229 @@
+open Core_kernel
```

Comment:
Introducing a new tier here above user-command is going to cause a lot of churn around archive-node, graphql, rosetta, cli, documentation and probably more.

Could we nest this under user-command? Or make it a sibling of user-command and fee-transfer and coinbase in trasnaction.ml instead?

Reply:
I think the easiest thing at this point would be to rename Command_transaction to User_command and rename User_command to Signed_transaction? Would that be ok?

Reply:
Yes, this seems okay!